Header header
int32 shutter
int32 gain
